### DISCUSSION TITLE
动态schema表报错

### DISCUSSION THREAD

**Discussion by kannon007** [apache/doris#18738] [A - General / Q&A] [answered]
文档上的动态schema表 demo sql 报错
![1681718067394](https://user-images.githubusercontent.com/10329981/232420981-566dfc38-de91-4c60-b6db-b99f39c45922.png)

```
execute sql: org.apache.doris.common.AnalysisException: errCode = 2, detailMessage = Syntax error in line 5:
               ...  -- ...标识该表是动态表， 是动态表的语法
               ^
Encountered: ...
Expected: IDENTIFIER
```

**Comment by zy-kkk** [comment, marked as answer]
What version of Doris are you using? Dynamic tables are a new feature in our upcoming 2.0 release
![image](https://user-images.githubusercontent.com/70795751/232444409-358b0131-33d0-4955-ae2d-b818440ee6a4.png)

**Comment by songyougang** [comment]
2.0.3建表还是报上面的错误，这个功能是移除了吗？
